Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17
+
+WORKDIR /
+
+COPY api api
+COPY cmd cmd
+COPY pkg pkg
+COPY test test
+COPY main.go main.go
+COPY Makefile Makefile
+COPY tools.go tools.go
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+RUN make build-cli
+
+CMD ./bin/combo run
```

Comment:
We'll need to update this to avoid specifying the `run` sub-command as it doesn't exist yet: how do you feel about the following structure instead:

```dockerfile
CMD ["/bin/combo"]
```

And the (future) controller deployment is responsible for providing the sub-command it wants to run as an argument?

Reply:
Yeah, I wasn't entirely sure what to include here without the `run` subcommand existing but this is a better idea.